tron/trondaemon.py

```diff
@@ -26,65 +26,43 @@
 log = logging.getLogger(__name__)
 
 
-class PIDFile(object):
-    """Create and check for a PID file for the daemon."""
+class LockFile(object):
+    """ Create and maintain a lock for the daemon """
 
     def __init__(self, filename):
         self.lock = flockfile.FlockFile(filename)
```

Comment:
i know this can seem like neverending task, but we're depending on 6 year old wrapper of a library deprecated 3 years ago.

can we ditch all of that and just use stdlib's fctnl?

docs: https://docs.python.org/3/library/fcntl.html#fcntl.flock

example: http://tilde.town/~cristo/file-locking-in-python.html

Reply:
Yeah, I can do that.